robot_skills/src/robot_skills/mockbot.py

```diff
@@ -193,8 +198,6 @@ def __init__(self, *args, **kwargs):
 
         self._person_names = []
 
-        self.learn_person = lambda name: True
```

Comment:
Removed but no-where added back

Reply:
This function is not part of the ED robot_part anymore. I did some cleaning of the mockbot, But the mockbot is not in sync with the rest at all.